### PR TITLE
Confluent Plugin

### PR DESCRIPTION
**Pull request by amolkokane-cirus**
Fixed a Null Pointer Exception issue
Added missing javadocs
Changed copyright to 2020

### CODE REVIEW

File: confluent-kafka-plugins/pom.xml [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+  <modelVersion>4.0.0</modelVersion>
+
+  <name>Confluent Kafka plugins</name>
+  <artifactId>confluent-kafka-plugins</artifactId>
+
+  <properties>
+    <assertj.version>3.12.0</assertj.version>
+    <awaitility.version>3.1.6</awaitility.version>
+    <confluent.version>5.3.1</confluent.version>
+<!--    <spark2.version>2.4.4</spark2.version>-->
```
Comment (CuriousVini) [inline confluent-kafka-plugins/pom.xml]: Could you please remove commented line?

File: kafka-plugins-common/src/main/java/io/cdap/plugin/batch/source/KafkaBatchConfig.java [hunk 2]
```diff
@@ -0,0 +1,379 @@
+package io.cdap.plugin.batch.source;
```
Comment (CuriousVini) [inline kafka-plugins-common/src/main/java/io/cdap/plugin/batch/source/KafkaBatchConfig.java:17]: missing copyright

File: kafka-plugins-common/pom.xml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (CuriousVini) [inline kafka-plugins-common/pom.xml:1]: missing copyright

**Review by CuriousVini — COMMENTED** [review]
Please include copyright to all the source code files

File: confluent-kafka-plugins/pom.xml (hunk 1 above)
Comment (shiva-cirus) [reply confluent-kafka-plugins/pom.xml]: Done

File: kafka-plugins-common/pom.xml (hunk 3 above)
Comment (shiva-cirus) [reply kafka-plugins-common/pom.xml:1]: Done

File: kafka-plugins-common/src/main/java/io/cdap/plugin/batch/source/KafkaBatchConfig.java (hunk 2 above)
Comment (shiva-cirus) [reply kafka-plugins-common/src/main/java/io/cdap/plugin/batch/source/KafkaBatchConfig.java:17]: Done

**Review by CuriousVini — APPROVED** [review]

**Pull request merged**